### ISSUE TITLE
Can't delete files over 100 Mb (7.0.4)

### ISSUE THREAD

**Issue by sbh01** [owncloud/core#12733]
I press on trashbin and see endless loading circle.
File is not deleting.
With small files all is ok.

btw, encryption is on on server.

Operating system: 10.0-RELEASE-p12 FreeBSD 10.0-RELEASE-p12
Web server:: Apache22-2.2.29_2
Database:: MySQL 5.5.40
PHP version:: PHP 5.3.29_2
ownCloud version:: 7.0.4.2 (stable)
Updated from an older ownCloud
List of activated apps:
-Calendar
-Contacts
-Deleted files
-External storage support
-First Run Wizard
-Full Text Search
-LDAP user and group backend
-Music
-PDF Viewer
-Pictures
-Share Files
-Text Editor
-Updater
-Versions
-Video Viewer
-User Photo
The content of config/config.php:
~~~
$CONFIG = array (
'instanceid' => 'xxxxx',
'datadirectory' => '/usr/local/www/owncloud/data',
'dbtype' => 'mysql',
'version' => '7.0.4.2',
'dbname' => 'owncloud',
'dbhost' => 'localhost',
'dbtableprefix' => 'oc_',
'dbuser' => 'oc_admin',
'installed' => true,
'theme' => '',
'maintenance' => false,
'ldapIgnoreNamingRules' => false,
'forcessl' => true,
'loglevel' => '3',
'maxZipInputSize' => 838860800,
'allowZipDownload' => true,
'trusted_domains' => 
array (
0 => 'cxx',
),
);
~~~
Are you using external storage, if yes which one::
-smb
Are you using encryption::
-yes
Browser:: Chrome 39.0.2171.65 m
Operating system:: Windows7 SP1 Professional

ownCloud Log:

~~~
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:24:20+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:24:21+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:25:06+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:25:07+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:26:09+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:26:10+00:00"}
{"app":"remote","message":"image not found: image:music.svg webroot: serverroot:\/usr\/local\/www\/owncloud","level":4,"time":"2014-12-10T04:35:26+00:00"}
{"app":"remote","message":"image not found: image:music.svg webroot: serverroot:\/usr\/local\/www\/owncloud","level":4,"time":"2014-12-10T04:35:27+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:35:47+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:35:48+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:35:54+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:35:54+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:36:17+00:00"}
{"app":"index","message":"OCP\\Files\\NotFoundException: ","level":4,"time":"2014-12-10T04:36:17+00:00"}
{"app":"PHP","message":"filemtime(): stat failed for \/usr\/local\/www\/owncloud\/data\/konko_am\/files\/VVT\/VMware-VMvisor-Installer-5.5.0.update02-2068190.x86_64.iso at \/usr\/local\/www\/owncloud\/lib\/private\/files\/storage\/local.php#133","level":3,"time":"2014-12-10T06:12:09+00:00"}
{"app":"Encryption library","message":"No share key found for user \"konko_am\" for file \"VMware-VMvisor-Installer-5.5.0.update02-2068190.x86_64.iso\"","level":2,"time":"2014-12-10T06:12:18+00:00"}
{"app":"webdav","message":"Sabre\\DAV\\Exception\\Forbidden: ","level":4,"time":"2014-12-10T06:12:32+00:00"}
{"app":"PHP","message":"filemtime(): stat failed for \/usr\/local\/www\/owncloud\/data\/konko_am\/files\/VVT\/VMware-VMvisor-Installer-5.5.0.update02-2068190.x86_64.iso at \/usr\/local\/www\/owncloud\/lib\/private\/files\/storage\/local.php#133","level":3,"time":"2014-12-10T06:48:22+00:00"}
{"app":"Encryption library","message":"No share key found for user \"konko_am\" for file \"VMware-VMvisor-Installer-5.5.0.update02-2068190.x86_64.iso\"","level":2,"time":"2014-12-10T06:48:36+00:00"}
{"app":"webdav","message":"Sabre\\DAV\\Exception\\Forbidden: ","level":4,"time":"2014-12-10T06:48:38+00:00"}
~~~

**Comment by jamfx** [comment]
Hi, I have the same issue on my ownlcoud. I wanted to delete a folder with 2,9GB of size, two files in it (each about 1,5GB) and I can not delete it. Upload was fine. But mine is / was shared! So I took away the Sharing and tried to delete again. But still I can not delte the folder. My files are not encrypted. Finaly I deleted them by FTP... Quick and dirty "workaround"...

**Comment by PVince81** [comment]
@jamfx when you tried to delete the folder/files, was there anything in owncloud.log or the web server log ?

It is still not clear what would cause this issue, so more information will help.

**Comment by jamfx** [comment]
@PVince81 I'm sorry. I had a look in the owncloud.log but there is nothing logged on that Date related to my files... What can I do now to help?

**Comment by sbh01** [comment]
I think that developers must do the same thing as we - upload a massive file to the oc and try to delete it.

**Comment by PVince81** [comment]
If only it were that simple...

I tried the following locally with 7.0.4:
1. Enable encryption
2. Upload a 143 MB file from the web UI
3. Delete the file
4. Check trashbin

The file was deleted properly.
Does it also happen if you disable 3rd party apps ? (like music)

**Comment by jamfx** [comment]
@PVince81 Please try BIGGER Files. Use Files like 1,5 GB of Size and put 2 of them in a folder. Then try to delete them. I don't have problems with "cute small" files like 200MB... :)
JamFX

**Comment by PVince81** [comment]
Haha. If it's that big like a few GB, then it's very likely you're running into a timeout.
Because files are not moved to trash, they are copied there... I heard there might be a fix in OC 8 that will fix that and move files when possible.

**Comment by sbh01** [comment]
Yep, i want to tell about bigger file but jamfx had outstripped me.

**Comment by PVince81** [comment]
Closing in favor of https://github.com/owncloud/core/issues/13974#issuecomment-77531761

Two possible causes: the trashbin expiration code is slow and triggers bogus rescans of the trashbin itself, making it slow.
Second possible cause: it copies files to trash instead of moving. This is fixed in 8.0.0.
